configuration/configuring-displays/en.md

```diff
@@ -3,20 +3,64 @@ title = "Configuring Displays"
 +++
 # Configuring Displays
 
-You can configure your display(s) by going to the Budgie Menu, opening the Settings app, and clicking the Displays section.
+With Solus it is easy to configure your display and setup how multiple displays will work together.
```

Comment:
[Should be "set up"](http://www.future-perfect.co.uk/grammar-tip/is-it-setup-set-up-or-set-up/).